indico/modules/events/templates/display/indico/_common.html

```diff
@@ -89,6 +89,7 @@
 
 {% macro render_datetime(prop_name, dt, timezone, format='medium') -%}
     <time itemprop="{{ prop_name }}" datetime="{{ dt.isoformat() }}">
+        {{ dt.strftime('%A') }}
```

Comment:
I'd rather `| format_date(...)` this so we keep the correct timezone and locale. Also make sure to test it with another language setting to make sure you get the weekday in that language.